### ISSUE TITLE
Interactive configuration handler

### ISSUE THREAD

**Issue by ElectricityMachine** [ElectricityMachine/SCR-SGPlus#34]
After #26, it might be nice to have a terminal configuration helper, whereby you press a button to enter config mode, and can remap keys. This would address the problem of user-configurable keybinds and allow us to operate exclusively on scancodes instead of [the keyboard module's canonical names](https://github.com/boppreh/keyboard/blob/d232de09bda50ecb5211ebcc59b85bc6da6aaa24/keyboard/_canonical_names.py), which would fix a potential issue if a user specifies "1" in addition to "Num 1", since they evaluate to the same thing.
